ci-operator/jobs/infra-periodics.yaml

```diff
@@ -306,7 +306,7 @@ periodics:
       - --confirm
       command:
       - /ko-app/commenter
-      image: gcr.io/k8s-prow/commenter:v20240731-a5d9345e59
+      image: us-docker.pkg.dev/k8s-infra-prow/images/commenter:v20240731-a5d9345e59
```

Comment:
we can't pull commenter from new location

Reply:
asked question upstream:
https://github.com/kubernetes/test-infra/pull/33206/files#r1741512119

Reply:
for now let's remove commenter upgrade from this PR